lua/smoothcursor/callbacks/init.lua

```diff
@@ -32,7 +31,15 @@ local function unplace_signs(with_timer_stop)
   if with_timer_stop == true then
     sc_timer:abort()
   end
-  vim.fn.sign_unplace('*', { buffer = vim.fn.bufname(), id = config.value.cursorID })
+
+  vim.fn.sign_unplace(
+    '*',
+    {
+      buffer = vim.fn.bufname(),
+      id = config.value.cursorID
+    }
+  )
+
```

Comment:
Would you mind formatting the code using `stylua`? It seems the change might not be necessary.

Reply:
should be formatted now